Source/Core/InputCommon/ControllerInterface/DInput/DInputJoystick.cpp

```diff
@@ -46,20 +46,18 @@ void InitJoystick(IDirectInput8* const idi8, HWND hwnd)
                   js_device->SetCooperativeLevel(nullptr, DISCL_BACKGROUND | DISCL_NONEXCLUSIVE)))
           {
             // PanicAlert("SetCooperativeLevel failed!");
-            js_device->Release();
             continue;
           }
         }
 
-        auto js = std::make_shared<Joystick>(js_device);
+        auto js = std::make_shared<Joystick>(js_device.Get());
```

Comment:
Then we should just pass it all the way through?  Doesn't make any sense to pass a raw pointer to put it back into a smart-ptr.